modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/liferay.js

```diff
@@ -123,7 +123,6 @@ Liferay = window.Liferay || {};
 
 			ioConfig.complete = function (response) {
 				if (
-					Object.keys(response).length > 0 &&
```

Comment:
Hi @NemethNorbert,

Have you checked if this condition was added for any reasons?

Regards.

Reply:
> Hey @georgel-pop-lr ,
> 
> It was added in the linked ticket that caused the regression, trying to check wether the response is ok or not. While in the second commit, we decided to rely on response.ok as well, this change remained in the code. I tested [LPS-142748](https://issues.liferay.com/browse/LPS-142748) as well with my changes, the issue is not reproducible.

Okay, let's see what the team has to say, from my part this PR looks good.

Regards.